shared/ff_util/include/ff_util/ff_names.h

```diff
@@ -382,6 +390,9 @@
 #define TOPIC_HARDWARE_IMU                          "hw/imu"
 #define TOPIC_HARDWARE_NAV_CAM                      "hw/cam_nav"
 #define TOPIC_HARDWARE_DOCK_CAM                     "hw/cam_dock"
+#define TOPIC_HARDWARE_CAM_SUFFIX_BAYER_RAW         "_bayer/image_raw"
```

Comment:
I think I understand the logic of why you needed to modify the previous bayer topic by appending `/image_raw` here. 

But there are existing bag recording profiles in other repos that refer to the old name, so we just need to make sure they get updated whenever this change takes effect.